pomegranate/bayes.pyx

```diff
@@ -804,8 +820,14 @@ cdef class BayesModel(Model):
         summaries /= summaries.sum()
 
         for i, distribution in enumerate(self.distributions):
-            if isinstance(distribution, DiscreteDistribution):
-                distribution.from_summaries(inertia, pseudocount)
+            if isinstance(
+                distribution,
+                (DiscreteDistribution, BernoulliDistribution, IndependentComponentsDistribution),
+            ):
+                distribution_smoothing = pseudocount
+                if alpha is not None:
```

Comment:
I'm not sure I like this. You're saying the parameter "pseudocount" and "alpha" will do the same thing here? I'd suggest having pseudocount consistent refer to the labels and alpha refer to the distributions alone, as you mentioned in the documentation.